frontend/src/components/FullEventList.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react'
+import { getAllEvents, getEventsById } from '../communications/userApi'
+import ListElements from '../components/listofevents/ListElements'
+import EventsListHeader from '../components/listofevents/EventsListHeader'
+import { useLocation } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+const FullEventList = (props) => {
+  const { t } = useTranslation()
+  const [events, setEvents] = useState()
+  const [error, setError] = useState()
+  const location = useLocation()
+
+  const getEvents = async () => {
+    const productId = location.state ? location.state : undefined
+    try {
+      if (productId === undefined) {
```

Comment:
és akkor itt is lehet picit egyszerűbben
```
if (!productId) {  // ha az első megoldás marad
```
ha a null-os akkor is jobb így ellenőrizni, mert nullra-undefre és üres stringre is false lesz.